### PR TITLE
chore: add issue templates

### PR DESCRIPTION
**Pull request by UrAvgDeveloper**

### CODE REVIEW

**Review by bryanchriswhite — APPROVED** [review]
Thanks @5A11!

**Comment by VijaySharma815** [comment]
@5A11 Please review this PR as Issue has been enabled on the repo.

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        Thank you for reporting the issue you are facing. Please complete the following so we can have the details necessary to assist you.
+  - type: checkboxes
+    id: prerequisites
+    attributes:
+      label: Prerequisites
+      description: Please confirm before submitting a new issue
+      options:
+        - label: I am running the [latest version of the Browser Extension Wallet](https://github.com/fetchai/fetch-wallet/releases).
+          required: true
+        - label: I checked the [documentation](https://docs.fetch.ai/browser-extension-wallet/) and found no answer to my problem.
```
Comment (VijaySharma815) [inline .github/ISSUE_TEMPLATE/bug-report.yml:18]: https://docs.fetch.ai/browser-extension-wallet/ : 404 - Not found

File: .github/ISSUE_TEMPLATE/feature-request.yml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    attributes:
+      value: |
+        Thank you for suggesting a new idea for this project. Please complete the following to ensure we have all the details to get things started.
+  - type: checkboxes
+    id: prerequisites
+    attributes:
+      label: Prerequisites
+      description: Please confirm before submitting a feature request.
+      options:
+        - label: I checked the [documentation](https://docs.fetch.ai/browser-extension-wallet/) and made sure this feature does not already exist.
```
Comment (VijaySharma815) [inline .github/ISSUE_TEMPLATE/feature-request.yml:16]: https://docs.fetch.ai/browser-extension-wallet/ : 404 - Not found

**Review by VijaySharma815 — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/feature-request.yml (hunk 2 above)
Comment (VijaySharma815) [reply .github/ISSUE_TEMPLATE/feature-request.yml:16]: for now we can use this one:
https://docs.fetch.ai/getting-started/how-to-use-browser-wallet/

File: .github/ISSUE_TEMPLATE/bug-report.yml (hunk 1 above)
Comment (VijaySharma815) [reply .github/ISSUE_TEMPLATE/bug-report.yml:18]: for now we can use this one:
https://docs.fetch.ai/getting-started/how-to-use-browser-wallet/

Comment (bryanchriswhite) [reply .github/ISSUE_TEMPLATE/bug-report.yml]: @VijaySharma815 I applied your suggestion but that also currently looks like a 404 to me.

Comment (VijaySharma815) [reply .github/ISSUE_TEMPLATE/bug-report.yml]: @bryanchriswhite we just merged some changes on docs so the updated URL should be:
https://docs.fetch.ai/basics/wallet/getting_started/

**Review by VijaySharma815 — APPROVED** [review]
LGTM!!

File: .github/ISSUE_TEMPLATE/bug-report.yml (hunk 1 above)
Comment (bryanchriswhite) [inline .github/ISSUE_TEMPLATE/bug-report.yml]:
```suggestion
        - label: I checked the [documentation](https://docs.fetch.ai/basics/wallet/getting_started/) and found no answer to my problem.
```

**Pull request merged**